src/PKSim.CLI.Core/MinimalImplementations/CLIDataImporter.cs

```diff
@@ -12,9 +14,15 @@ public IEnumerable<DataRepository> ImportDataSets(IReadOnlyList<MetaDataCategory
          return Enumerable.Empty<DataRepository>();
       }
 
-      public DataRepository ImportDataSet(IReadOnlyList<MetaDataCategory> metaDataCategories, IReadOnlyList<ColumnInfo> columnInfos, DataImporterSettings dataImporterSettings)
+      public IEnumerable<DataRepository> ImportFromConfiguration(ImporterConfiguration configuration, bool promptForConfirmation, IReadOnlyList<MetaDataCategory> metaDataCategories,
+         IReadOnlyList<ColumnInfo> columnInfos, DataImporterSettings dataImporterSettings)
       {
-         return new NullDataRepository();
+         return Enumerable.Empty<DataRepository>();
+      }
+
+      (IEnumerable<DataRepository> DataRepositories, ImporterConfiguration Configuration) IDataImporter.ImportDataSets(IReadOnlyList<MetaDataCategory> metaDataCategories, IReadOnlyList<ColumnInfo> columnInfos, DataImporterSettings dataImporterSettings)
```

Comment:
why do you need explicit implementation here?`IDataImporter.`

Reply:
We are keeping the same logic as before (with the old importer), this is a dependency of other service classes so from the cli it needs an UIComponents-free implementation that can be instantiated from cli. In the future this should be a real implementation so the importer could be used from R for example but I would say this is a future feature and only if needed.